app/views/spree/admin/shared/_transaction_sync_log_table.html.erb

```diff
@@ -16,6 +17,7 @@
         <td><%= log.id %></td>
         <td><%= link_to log.order.number, spree.edit_admin_order_path(log.order) %></td>
         <td><%= log.order_transaction&.transaction_id || "-" %></td>
+        <td><%= log.refund_transaction&.transaction_id || "-" %></td>
```

Comment:
Would this ever have both a `log.order_transaction` and a `log.refund_trasnaction`?